Configuration/config.py

```
@@ -0,0 +1,16 @@
+# firebase admin 
+import firebase_admin
+# credentials,auth,firestore 
+from firebase_admin import credentials,auth,firestore 
+
+    
+class Configuration:
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)